.github/actions/rockylinux-build/entrypoint.sh

```diff
@@ -16,10 +16,10 @@ dnf install -y epel-release git make wget 'dnf-command(config-manager)' && \
 git clone --depth 1 -b $1 https://github.com/ImageMagick/ImageMagick.git
 
 # Generate updated .src.rpm
-cd ImageMagick && \
+ cd ImageMagick && \
```

Comment:
please pay attention to formatting

```suggestion
cd ImageMagick && \
```